src/test/java/io/github/oliviercailloux/y2018/apartments/apartment/json/JsonConvertTest.java

```diff
@@ -84,12 +98,37 @@ void jsonToApartmentsTest() throws IOException, URISyntaxException {
             .setTele(false)
             .build());
 
-    URI ressource = JsonConvertTest.class.getResource("jsonApartments.json").toURI();
-    Path jsonPath = Path.of(ressource);
-
-    List<Apartment> apartmentsTest = JsonConvert.jsonToApartments(jsonPath);
+    List<Apartment> apartmentsTest = JsonConvert.jsonToApartments(this.jsonTestPath);
 
     assertEquals(apartmentsRef.get(0).hashCode(), apartmentsTest.get(0).hashCode());
     assertEquals(apartmentsRef.get(1).hashCode(), apartmentsTest.get(1).hashCode());
   }
+
+  /** Test if the adapter method works */
+  @Test
+  void getAdapterTest() throws Exception {
+    JsonbConfig defaultConfig = new JsonbConfig();
+    JsonbAdapter<Apartment, JsonObject> adapter = JsonConvert.getAdapter();
+    JsonbConfig config = new JsonbConfig().withAdapters(adapter);
+    assertNotEquals(defaultConfig, config);
```

Comment:
On se disperse : ceci n’est pas ce que vous voulez tester. Il ne faut pas tester les API que vous utilisez si vous pouvez raisonnablement considérer que vous pouvez leur faire confiance (ce qui est le cas ici), ou il faut en faire un travail en soi, à séparer du reste. OPT

Reply:
Ce test permettait de vérifier que `JsonConvert.getAdapter()` était bien implémentée et était bien différent de l'adapter normal.
On l'enlève

Reply:
Fix in 2fd4cc3729016854db9dc872f869b9b221bd4ac2